app/models/curator_scope.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
```

Comment:
I would suggest that we start putting documentation at the top of models that explains the model, uses, data contained at a higher level.  this would be super for future devs, for the code to be more self documenting.